<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectionSettings</class>
 <widget class="QWidget" name="connectionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="io_groupBox">
     <property name="title">
      <string>IO Node</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>hostname</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="io_hostname">
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>port</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="io_port">
           <property name="text">
            <string>6678</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="io_connect">
        <property name="text">
         <string>browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="fileView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="processing_groupBox">
     <property name="title">
      <string>Processing Node</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QPushButton" name="proc_connect">
        <property name="text">
         <string>render</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>hostname</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="proc_hostname">
           <property name="text">
            <string>127.0.0.1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="proc_port">
           <property name="text">
            <string>8678</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>resolution</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="proc_width"/>
         </item>
         <item>
          <widget class="QLineEdit" name="proc_height"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
